src/snowflake/snowpark/modin/plugin/_internal/index.py

```diff
@@ -177,6 +177,8 @@ def __getattr__(self, key: str) -> Any:
             if not key.startswith("_"):
                 native_index = native_pd.Index([])
                 if hasattr(native_index, key):
+                    # Any methods that not supported by the current Index.py but exist in a
```

Comment:
comment Yun requested in https://github.com/snowflakedb/snowpark-python/pull/1751